tests/php/authentication/class-test-jwt-auth.php

```diff
@@ -117,14 +117,16 @@ public function test_generate_token() {
 		/**
 		 * Test if error is thrown when SECURE_AUTH_KEY is not defined.
 		 */
-		$token = $this->jwt_auth->generate_token( $rest_request );
-		$this->assertTrue( is_wp_error( $token ) );
-		$this->assertEquals( 'rest_auth_key', $token->get_error_code() );
+		if ( ! defined( 'SECURE_AUTH_KEY' ) ) {
+			$token = $this->jwt_auth->generate_token( $rest_request );
```

Comment:
@valendesigns is there any point in keeping these tests wrapped in here?

Reply:
So if it's not defined we know it complains and is tested locally, because we are 100% not defining it. We could look into why Travis is setting it in the first place but that's a rabbit hole I don't feel is worth the effort.